values.yaml

```diff
@@ -87,14 +89,14 @@ global:
     # dataStores: List of stores used by data-service and associated data-readers to read resources and return to a client
     # nb. the mountPath here will be referenced by all client resource requests
     dataStores:
-      palisade-data-store:
```

Comment:
The concept here is that the data mounts are a map so that a given service can mount more than one storage if required.